src/app/radar-chart/radar-chart.component.ts

```diff
@@ -7,9 +9,15 @@ import { Component, OnInit } from '@angular/core';
 })
 export class RadarChartComponent implements OnInit {
 
-  constructor() { }
+  // a component can have its own clock
+  update = interval(1000);
```

Comment:
I'm using a local interval observer to check that data inside cityio:CityioServer, but we can directly subscribe using `this.cityio.update.subscribe(()=>{})`